hack/e2e.sh

```diff
@@ -57,6 +57,10 @@ Examples:
 
     See https://onsi.github.io/ginkgo/ for more ginkgo options.
 
+3) reuse the cluster and don't tear down it after the testing
+
+    REUSE_CLUSTER=y SKIP_DOWN=y ./hack/e2e.sh -- <e2e args>
+
```

Comment:
Suggest adding examples for how to set *_IO_MIRROR

Reply:
done in 8f84be249f96aa8e8ae53538b26c7ea1e11adce6